### ISSUE TITLE
Important

### ISSUE THREAD

**Issue by Tooconfident** [yutingong/f1-3-c2p1-colmar-academy#1]
https://github.com/yutingong/f1-3-c2p1-colmar-academy/blob/master/ColmarAcademy/index.html#L7

`<link href="file:///Users/yutingong/ColmarAcademy/Resources/css/style.css" type="text/css" rel="stylesheet">`

Since this is your 3rd project being submitted for review, by this time it should have been addressed that you need to be using relative paths instead of your local computer path to link your stylesheet. The above link is incorrect and I cannot properly review your work until I change your path.

basically for everywhere in your html where you have **file:///Users/yutingong/ColmarAcademy**, you need to replace that with a period:

`<link href="./Resources/css/style.css" type="text/css" rel="stylesheet">`
`<img src="./Resources/capstone_colmar_assets/images/thesis-now-and-then.jpg"/>`
